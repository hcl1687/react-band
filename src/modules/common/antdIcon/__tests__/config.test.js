import configFactory from '../config'

const config = configFactory()

describe('common/antdIcon/config', () => {
  it('should render correctly', () => {
    expect(config).toMatchSnapshot()
  })
})
