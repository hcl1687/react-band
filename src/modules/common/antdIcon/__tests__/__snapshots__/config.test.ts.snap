// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common/antdIcon/config should render correctly 1`] = `
Object {
  "name": "antdIcon",
}
`;
